prometheus-client-derive/src/registrant/attribute.rs

```diff
@@ -0,0 +1,135 @@
+use proc_macro2::Span;
+use quote::ToTokens;
+use syn::spanned::Spanned;
+
+// do not derive debug since this needs "extra-traits"
+// feature for crate `syn`, which slows compile time
+// too much, and is not needed as this struct is not
+// public.
+#[derive(Default)]
+pub struct Attribute {
+    pub help: Option<syn::LitStr>,
```

Comment:
Shall we make the `help` as a required field? Lack of help message might make no sense.